src/newcharacter.cpp

```diff
@@ -2747,6 +2768,13 @@ tab_direction set_description( avatar &you, const bool allow_reroll,
                         you.blood_rh_factor = false;
                     }
                     break;
+                case char_creation::GENDER:
+                    if( you.male ) {
+                        you.male = false;
+                    } else {
+                        you.male = true;
+                    }
```

Comment:
Why not like this?
```suggestion
                    you.male = !you.male;
```

Reply:
Thanks. Of course you're absolutely right.